manufacturing-ai-suite/wind-turbine-anomaly-detection/time_series_analytics_microservice/udfs/windturbine_anomaly_detector.py

```diff
@@ -148,72 +148,84 @@ def process_the_point(x,y):
             return 1
         # extract the wind speed and power from the point
         for point_data in point.fieldsDouble:
-            if point_data.key == self.x_name:
-                x = point_data.value
-            elif point_data.key == self.y_name:
-                y = point_data.value
+            if point_data.key == "Pressure":
+                pressure = point_data.value
+            elif point_data.key == "CO2 Weld Flow":
+                co2_weld_flow = point_data.value
+            elif point_data.key == "Feed":
+                feed = point_data.value
+            elif point_data.key == "Primary Weld Current":
+                primary_weld_current = point_data.value
+            elif point_data.key == "Wire Consumed":
+                wire_consumed = point_data.value
+            elif point_data.key == "Secondary Weld Voltage":
+                secondary_weld_voltage = point_data.value
             else:
                 continue
+
+        logger.info(f"Pressure: {pressure}, CO2 Weld Flow: {co2_weld_flow}, Feed: {feed}, \
+                    Primary Weld Current: {primary_weld_current}, Wire Consumed: {wire_consumed}, \
+                        Secondary Weld Voltage: {secondary_weld_voltage}")
```

Comment:
Hey @pooja-intel ,
Thanks for quickly checking the ingestion part.
As you are working on understanding the spine app changes, you can introduce these changes as configs for the docker compose deployment of that spine app (will see on the feasibility later) if it's easily doable.

Post that you can prioritize the udf integration and the helm deployment stuff you can take it later.